### ISSUE TITLE
remove learner profiles

### ISSUE THREAD

**Issue by ErinBecker** [carpentries/carpentries.org-archive#13]
The [learner profiles](https://software-carpentry.org/audience/) page is currently listed in both the `Learn` and `Teach` drop down menus. There is no context on this page for what the Learner Profiles represent. I expect it will be unclear what the significance of these profiles is. To my knowledge, these were not displayed from any nav menu on the SWC site. 

It may be valuable to include these learner profiles on the site somewhere, but if so we should add some information that contextualizes these and helps people use this information. For example, if these are profiles of people who would benefit from a SWC (or DC?) workshop, that could be used to help people make decisions about whether to request a workshop. If there would be different learner profiles for the ideal SWC vs DC audience, we should indicate that somehow. 

Long story short, let's keep an issue open about re-targeting these learner profiles and making them a useful resource for people in deciding whether or not to request / attend / teach a workshop, but remove them from the site until we are clearer about what they are for.

**Comment by weaverbel** [comment]
They were on the SWC site and in a menu as Our Audience. I felt Learner Profiles better captured what they actually are. We do use these in instructor training so what is your plan if they aren't listed anywhere?

**Comment by weaverbel** [comment]
I have removed from one menu. Reluctant to remove it altogether but wonder if these really belong in instructor training in some way. I will link to it from the learn landing page in case it has to be removed from the menu,

**Comment by ErinBecker** [comment]
This content has been migrated to this repo (https://carpentries.org/audience/) and is now included as a link within the Learn page.
